### PR TITLE
CONFIG: FET-202 microsoft-contrib tslint rules

### PR DESCRIPTION
**Pull request by ivanmjartan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: typescript/tslint.js [hunk 1]
```diff
@@ -29,6 +30,21 @@ module.exports = {
...
+        "no-string-based-set-interval": true, // https://github.com/Microsoft/tslint-microsoft-contrib/wiki/TSLint-and-the-Microsoft-Security-Development-Lifecycle
+        "no-string-based-set-timeout": true, // https://github.com/Microsoft/tslint-microsoft-contrib/wiki/TSLint-and-the-Microsoft-Security-Development-Lifecycle
+        "no-eval": true, // https://github.com/Microsoft/tslint-microsoft-contrib/wiki/TSLint-and-the-Microsoft-Security-Development-Lifecycle
+        "no-document-write": true, // https://github.com/Microsoft/tslint-microsoft-contrib/wiki/TSLint-and-the-Microsoft-Security-Development-Lifecycle
+        "no-http-string": true, // https://github.com/Microsoft/tslint-microsoft-contrib/wiki/TSLint-and-the-Microsoft-Security-Development-Lifecycle
+        "no-inner-html": true, // https://github.com/Microsoft/tslint-microsoft-contrib/wiki/TSLint-and-the-Microsoft-Security-Development-Lifecycle
+        "react-no-dangerous-html": true // https://github.com/Microsoft/tslint-microsoft-contrib/wiki/TSLint-and-the-Microsoft-Security-Development-Lifecycle
```
Comment (no23reason) [inline typescript/tslint.js]: I think some rules are missing:
no-function-constructor-with-string-args
no-reserved-keywords

**Comment by no23reason** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gdc-js-style-merge-compliance-check/7/ : SUCCESS in 38s

**Pull request merged**